.github/workflows/test.yaml

```diff
@@ -60,3 +60,29 @@ jobs:
           coverage-path: ./files/success/test_coverage_cobertura.xml
           coverage-type: cobertura
           coverage-threshold: 44
+      - name: Test Hide Success
+        uses: ./
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          comment-title: Test Hide Success
+          results-path: ./files/**/*.trx
+          allow-failed-tests: true
+          only-show-failed-tests : true
+      - name: Test Hide Output
+        uses: ./
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          comment-title: Test Hide Output
+          results-path: ./files/**/test_result.trx
+          allow-failed-tests: true
+          only-show-failed-tests: false
+          show-test-output: false
+      - name: Test Show Output
+        uses: ./
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          comment-title: Test Show Output
+          results-path: ./files/**/test_result.trx
+          allow-failed-tests: true
+          only-show-failed-tests: false
+          show-test-output: true
```

Comment:
Please add a new line at the end of this file. 
Also, README needs to be updated accordingly because it is what everyone sees on [the marketplace page](https://github.com/marketplace/actions/dotnet-test-reporter)

Reply:
done